### PR TITLE
Add migration for adding Fedora4 record identifiers

### PR DESCRIPTION
**Pull request by bkeese**
References #1954

### CODE REVIEW

File: db/migrate/20170609132731_add_fedora4_record_identifiers.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class AddFedora4RecordIdentifiers < ActiveRecord::Migration
+
+  def up
+    say_with_time("Add Fedora4 record identifers") do
+      MediaObject.find_each({},{batch_size:5}) do |mo|
+        say("MediaObject: #{mo.id}, URI: #{mo.uri}")
+        if mo.descMetadata.record_identifier.empty?
+          mo.descMetadata.ensure_identifier_exists! mo.uri
```
Comment (cjcolvar) [inline db/migrate/20170609132731_add_fedora4_record_identifiers.rb]: Do you really need this?  Wouldn't a mo.save be sufficient to trigger the callback which runs ensure_identifier_exists!?

**Comment by bkeese** [comment]
Yep. Fixed.

**Comment by cjcolvar** [comment]
I'm wondering if we should just make this a rake task instead of a DB migration.  I feel like that is the approach taken by sufia and what we did last time (https://github.com/avalonmediasystem/avalon/blob/develop/lib/tasks/avalon.rake#L131-L144).  That way this doesn't bog down everyone who doesn't need to run it and it fixes the problem with the build as a bonus!

**Comment by bkeese** [comment]
Changed to rake task.

**Pull request merged**